qiita_db/metadata_template/test/test_sample_template.py

```diff
@@ -1528,6 +1540,29 @@ def test_extend_add_samples(self):
         md_ext = pd.DataFrame.from_dict(md_dict, orient='index')
 
         st.extend(md_ext)
+        analysis = Analysis(1)
```

Comment:
Can you actually create a different test for this? It would be easier to identify which functionality is missing.